app.js

```diff
@@ -0,0 +1,139 @@
+    mobiletech: ['Ruby on rails', 'CSS', 'JavaScript'],
+    live_version: 'https://svitalis123.github.io/portfolio/app',
+    source_link: 'https://github.com/svitalis123/portfolio',
+    head: 'Keeping track of hundreds of components',
+    para: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  },
+  {
+    id: '5',
+    img: './assets/main/projectdetails.jpg',
+    deskimg: './assets/main/port.jpg',
+    head: 'Keeping track of hundreds of components',
+    desktech: ['Codekit', 'GitHub', 'Javasript', 'Bootstrap', 'Terminal', 'Codepen'],
+    mobiletech: ['Ruby on rails', 'CSS', 'JavaScript'],
+    live_version: 'https://svitalis123.github.io/portfolio/app',
+    source_link: 'https://github.com/svitalis123/portfolio',
+    para: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  },
+  {
+    id: '6',
+    img: './assets/main/projectdetails.jpg',
+    deskimg: './assets/main/port.jpg',
+    head: 'Keeping track of hundreds of components',
+    desktech: ['Codekit', 'GitHub', 'Javasript', 'Bootstrap', 'Terminal', 'Codepen'],
+    mobiletech: ['Ruby on rails', 'CSS', 'JavaScript'],
+    live_version: 'https://svitalis123.github.io/portfolio/app',
+    source_link: 'https://github.com/svitalis123/portfolio',
+    para: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  },
+];
+
+function getId(id) {
+  if (window.innerWidth > 920) {
+    const challdesk = (str) => {
+      const closebtn2 = document.getElementById(`${str}`);
+      closebtn2.addEventListener('click', () => {
+        const divDetails2 = document.querySelector('#project1amend1desc');
+        divDetails2.classList.remove('details_ammenddescreverse');
+      });
+    };
+    for (let c = 0; c < data.length; c += 1) {
```

Comment:
- [ ] [OPTIONAL] It's a good idea to use .forEach() when it's possible instead of for loop, this will make your code follow best practices